src/movie.c

```diff
@@ -95,6 +95,7 @@
 #ifdef WIN32
 #include <windows.h>
 #endif
+#include "gmt_gsformats.h"
```

Comment:
I couldn't really figure out based on the other modules if there is a convention for ordering the header files relative to each other and the macros, so let me know if this isn't the right place for this line.